File: solutions/1720-crawler-log-folder/solution.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        c=0
        for i in logs:
            if i=="../":
                c-=1
                c=0 if c<0 else c
            elif i=="./":
                c=c
            else:
                c+=1
        return c

File: solutions/1720-crawler-log-folder/solution.java
class Solution {
    public int minOperations(String[] logs) {
        int c=0;
        for(String i:logs)
        {
            if(i.equals("../"))
            {
                c-=1;
                c=c<0?0:c;
            }
            else if(i.equals("./"))
                c=c;
            else
                c+=1;
        }
        return c;
    }
}
